.github/workflows/ci.yml

```diff
@@ -15,57 +15,35 @@ on:
   pull_request:
 
 jobs:
-  test-cpython:
+  test-python:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        # Commented out as tests with tox on python3.9 fail with updated dependencies.
-        # See https://github.com/pypiserver/pypiserver/issues/406.
-        # uncomment when the issue #406 is resolved.
-        # python-version: ["3.6", "3.7", "3.8", "3.9"]
-        python-version: ["3.6", "3.7", "3.8"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "pypy3.9"]  # "3.11-dev"
```

Comment:
Oups, somehow the comment here didn't get sent:
Thanks a lot for the added versions! But why do we remove `3.6` from the matrix? I feel it would be useful to run the CI on this version still too. But what do you think?